src/platform/Kconfig

```diff
@@ -291,13 +307,13 @@ config MAX_CORE_COUNT
 	int
 	default 2 if APOLLOLAKE
 	default 4 if ICELAKE || CANNONLAKE || SUECREEK || TIGERLAKE
+	default 3 if METEORLAKE
 	default 1
 	help
 	  Maximum number of cores per configuration
 
 config CORE_COUNT
 	int "Number of cores"
-	default MP_NUM_CPUS if KERNEL_BIN_NAME = "zephyr"
```

Comment:
why this? Doesn't this break or at least needlessly change other platforms?

Reply:
It was a workaround, zephyr had incorrect number of cores